modules/navier_stokes/test/tests/ins/mms/supg/supg_pspg_adv_dominated_mms.i

```diff
@@ -0,0 +1,266 @@
+mu=1.5e-4
+rho=2.5
+
+[GlobalParams]
+  gravity = '0 0 0'
+  supg = true
+  pspg = true
+  convective_term = true
+  integrate_p_by_parts = false
+  transient_term = true
+  laplace = true
+  u = vel_x
+  v = vel_y
+  p = p
+  alpha = 1e0
+  order = FIRST
+  family = LAGRANGE
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1.0
+  ymin = 0
+  ymax = 1.0
+  elem_type = QUAD9
```

Comment:
This problem should use QUAD4s, right?. Besides this and the `pspg=true` parameter, are there any other differences between the two input files? Seems like they should just be combined into one and the tests controlled with CLI args.